include/nestegg/nestegg.h

```diff
@@ -439,6 +452,18 @@ int nestegg_packet_encryption(nestegg_packet * packet);
 int nestegg_packet_iv(nestegg_packet * packet, unsigned char const ** iv,
                       size_t * length);
 
+/** Query the packet for offsets.
+@param packet            Packet initialized by #nestegg_read_packet.
+@param partition_offsets Storage for queried offsets.
+@param num_offsets       Length of returned offsets, may be 0.
```

Comment:
I think it can legitimately be 0, it would be weird but it could. I should probably be returning an error if the packet isn't partitioned though.